addons/repair/models/repair.py

```diff
@@ -279,7 +279,7 @@ def action_repair_confirm(self):
         return True
 
     def action_repair_cancel(self):
-        if self.state == 'done':
+        if self.filtered(lambda repair: repair.state == 'done'):
```

Comment:
small code improvement

```suggestion
        if any(repair.state == 'done' for repair in self):
```